lpcompiler.pas

```diff
@@ -1549,6 +1550,36 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
     end;
   end;
 
+  procedure AddDirectiveHint(Tok: EParserToken);
+  var
+    Message: String = '';
+  begin
+    if (Tokenizer.Expect([tk_typ_String, tk_sym_SemiColon]) = tk_typ_String) then
+      Message := Copy(Tokenizer.TokString, 2, Tokenizer.TokLen - 2);
+
+    with TLapeType_Method(Result.Method.VarType) do
+      case Tok of
+        tk_kw_Deprecated:
+          begin
+            IsDeprecated := True;
```

Comment:
What about double deprecation directives? Just assume the last directive is correct? Or maybe error?